Angular/Drawer template styling

vegeorgi opened the pull request.

vegeorgi committed:
Angular - Drawer template styling

vegeorgi committed:
Add icon class

vegeorgi committed:
Remove unused code

vegeorgi committed:
Rename drawer 'current' property with 'selectedPage'

vegeorgi committed:
Remove custom styles for the Action bar component

MartoYankov commented on Angular/DrawerNavigation/app/browse/browse.component.html:
```diff
@@ -1,11 +1,16 @@
     <StackLayout tkDrawerContent>
         <StackLayout>
-            <MyDrawer></MyDrawer>
+                <MyDrawer [selectedPage]="'Browse'"></MyDrawer>
```
a minor indentation problem? It's on every page.

vegeorgi replied:
Done :)

MartoYankov commented on Angular/DrawerNavigation/app/shared/drawer.component.html:
```diff
@@ -1,9 +1,20 @@
+                <GridLayout rows="*" columns="auto, *"
+                    (tap)="navigateToRoute(item.route)"
+                    class="sidedrawer-list-item" 
+                    [class.selected]="item.title===selectedPage">
```
According to Angular guidelines, this shouldn't be here. There should be no presentation logic in the markup.

"Do put presentation logic in the component class, and not in the template."

vegeorgi replied:
Done :)

MartoYankov reviewed (changes requested):
bump

vegeorgi committed:
Fixes after MIYankov comments

1.Remove indentation in the page templates
2. Move the logic from the markup to the component

vegeorgi committed:
Fix typo

vegeorgi committed:
Fix sass files structure
